backend/src/govsg/middlewares/govsg-template.middleware.ts

```diff
@@ -16,20 +16,37 @@ import { CampaignService, StatsService, UploadService } from '@core/services'
 import { GovsgTemplatesAccess } from '@govsg/models'
 import { CampaignGovsgTemplate } from '@govsg/models/campaign-govsg-template'
 import { GovsgMessage } from '@govsg/models/govsg-message'
-import { GovsgTemplate } from '@govsg/models/govsg-template'
+import {
+  GovsgTemplate,
+  GovsgTemplateLanguageMetadata,
+} from '@govsg/models/govsg-template'
 import { GovsgTemplateService } from '@govsg/services'
 import { NextFunction, Request, Response } from 'express'
 
 const logger = loggerWithLabel(module)
 
+const convertMultilingualSupportToResponse = (
+  multilingualSupport: Array<GovsgTemplateLanguageMetadata>
+) => {
+  return multilingualSupport.map((languageSupport) => ({
+    body: languageSupport.body,
+    language: languageSupport.language,
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    code: languageSupport.languageCode ?? languageSupport['language_code'],
+  }))
+}
+
 function convertGovsgTemplateToResponse(template: GovsgTemplate) {
   return {
     id: template.id,
     body: template.body,
     params: template.params,
     param_metadata: template.paramMetadata,
     name: template.name,
-    multilingual_support: template.multilingualSupport,
+    languages: convertMultilingualSupportToResponse(
```

Comment:
Too much confusion between camel case and snake case on fields and nested fields. Troubleshooting this was extremely painful. Chose to avoid this problem altogether.